File: code/backend/java/core/src/main/java/games/paths/core/model/story/ClassBonusInfo.java
package games.paths.core.model.story;

/**
 * ClassBonusInfo - Domain model for a single class bonus row.
 * Nested under ClassInfo to expose the {@code list_classes_bonus} rows on the
 * public API. Immutable record.
 */
public record ClassBonusInfo(String uuid, String statistic, int value) {
}

File: code/backend/php/src/Core/Domain/Story/ClassBonusInfo.php
<?php

declare(strict_types=1);

namespace Games\Paths\Core\Domain\Story;

class ClassBonusInfo
{
    public function __construct(
        public ?string $uuid = null,
        public ?string $statistic = null,
        public int $value = 0
    ) {
    }
}
